.github/workflows/package.yml

```diff
@@ -0,0 +1,53 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright 2020 - 2024 Pionix GmbH and Contributors to EVerest
+
+name: Package EVerest Admin Panel
+
+on:
+  pull_request:
+    branches:
+      - main
+      - spr/main/*
+  push:
+    branches:
+      - main
+    tags:
+      - '*'
+  workflow_dispatch: {}
+
+jobs:
+  package:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Use Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: '18.x'
+      - uses: pnpm/action-setup@v2
+        name: Install pnpm
+        with:
+          version: 8
+          run_install: false
+      - name: Get pnpm store directory
+        shell: bash
+        run: |
+          echo "STORE_PATH=$(pnpm store path --silent)" >> $GITHUB_ENV
+      - uses: actions/cache@v3
+        name: Setup pnpm cache
+        with:
+          path: ${{ env.STORE_PATH }}
+          key: ${{ runner.os }}-pnpm-store-${{ hashFiles('**/pnpm-lock.yaml') }}
+          restore-keys: |
+            ${{ runner.os }}-pnpm-store-
+      - name: Install dependencies
+        run: pnpm install
+      - name: Build
+        run: pnpm build
+      - name: Tar
+        run: tar -C dist -czf everest-admin-panel.tar.gz ./
+      - name: Release
```

Comment:
We should maybe upload the result as an artifat if it is not released, so that it can be downloaded from the CI run

Reply:
@nzbr This could be a new issue but is not part of this commit. This commit is about splitting up the ci and making a working concurrency limit for deploy. We never needed this so far though and it only eats up artifact storage so I don't think it is important right now. For reviewing stuff we already have github pages where you get the build-files as well if you just go to the appropriate branch

Reply:
I see. In that case, the tar step should probably be skipped as well, because it will just take up unnecessary ci time (though these things aren't _really_ a huge concern here, as the repo is public)